routes/mentor.go

```diff
@@ -3,8 +3,45 @@ package routes
 import (
 	"fmt"
 	"net/http"
+	"encoding/json"
+	"io/ioutil"
+	"github.com/jinzhu/gorm"
+	"kwoc20-backend/models"
 )
 
 func MentorReg(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "hello")
+
+	var mentor models.Mentor
+	body, _ := ioutil.ReadAll(r.Body)
+	err := json.Unmarshal(body, &mentor)
+	if err != nil {
+	  http.Error(w, err.Error(), 400)
+	  return 		
+	}
+
+	
+	db, err := gorm.Open("sqlite3", "kwoc.db")
+	if err != nil {
+		 http.Error(w, err.Error(), 500)
+		 return
+	}
+	defer db.Close()
+
+	err = db.Create(&models.Mentor{
+				Name: Mentor.Name,
```

Comment:
above at line number 15 , u declared mentor with small "m" and here u are using capital M